Multicolored waypoints

De1taVee opened the pull request.

signifatee committed:
Added new column "point_type" in database

signifatee committed:
Removed extra lines

signifatee committed:
Removed extra lines

signifatee committed:
Added multicolored waypoints;
Added type field, added type check;
Waypoint images now .png, correlated package was renamed;
Added type row in database;

ptz2018 commented on src/js/App.js:
```diff
@@ -68,13 +72,13 @@ function App() {
             />
             { groups && groups.length > 0 && <CheckboxList groups={groups} onChange={onSelectGroups}></CheckboxList>}
 
+
```
please remove

ptz2018 commented on src/js/App.js:
```diff
@@ -68,13 +72,13 @@ function App() {
-                    <RStyle.RStyle>
-                        <RStyle.RIcon src={locationIcon} anchor={[0.5, 0.8]} className="map__icon"/>
-                    </RStyle.RStyle>
+
```
please remove

ptz2018 commented on src/js/App.js:
```diff
@@ -68,13 +72,13 @@ function App() {
+
                     {
                         selectedGroups.map(g => g.points.map(p =>
+
```
please remove

ptz2018 commented on src/js/App.js:
```diff
@@ -85,11 +89,16 @@ function App() {
                                 }
                             >
+                            <>
+                            <RStyle.RStyle>
```
please amend indentation

ptz2018 commented on src/js/config.js:
```diff
@@ -1,2 +1,2 @@
-const BASE_URL = 'http://localhost:8090';
+const BASE_URL = 'http://localhost:8080';
```
please revert from commit

ptz2018 commented on src/main/java/com/example/projects/dto/PointDTO.java:
```diff
@@ -1,19 +1,28 @@
     }
 
-    public PointDTO(int id, double x, double y) {
+    public PointDTO(int id, double x, double y, PointTypeEnum point_type) {
```
pointType

ptz2018 commented on src/main/java/com/example/projects/dto/PointDTO.java:
```diff
@@ -1,19 +1,28 @@
+        this.point_type = point_type;
     }
 
+    public PointTypeEnum getPoint_type() { return point_type; }
```
pointType

ptz2018 commented on src/main/java/com/example/projects/model/Point.java:
```diff
@@ -13,6 +14,8 @@ public class Point {
     private double x;
     private double y;
+    @Enumerated(EnumType.STRING)
+    private PointTypeEnum point_type;
```
pointType

ptz2018 commented on src/main/resources/static/png/point.svg:
why do you need this?

signifatee committed:
Renamed point_type to pointType;
Removed extra lines;
Removed flyway.properties due security problem;
Removed not used files;

signifatee committed:
Renamed point_type to pointType;
Removed extra lines;
Removed flyway.properties due security problem;
Removed not used files;

signifatee committed:
Renamed point_type to pointType;
Removed extra lines;
Removed flyway.properties due security problem;
Removed not used files;

De1taVee committed:
config reverted
